_gsocproposals/2021/proposal_cling_dev.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Improve Cling’s Development Lifecycle
+layout: gsoc_proposal
+project: IRIS-HEP
+year: 2021
+organization: unl
+---
+
+Improve Cling’s Development Lifecycle
+
+## Description
+
+Cling is an interactive C++ interpreter, built on top of Clang and LLVM compiler infrastructure. Cling realizes the read-eval-print loop (REPL) concept, in order to leverage rapid application development. Implemented as a small extension to LLVM and Clang, the interpreter reuses their strengths such as the praised concise and expressive compiler diagnostics.
+
+
+## Task ideas and expected results
+
+The project foresees to enhance the github actions infrastructure by adding development process automation tools:
+ * Code Coverage information (codecov)
+ * Static code analysis (clang-tidy)
+ * Coding conventions checks (clang-format)
+ * Release binary upload automation
+
+The student should be prepared to write a progress report and present the results.
+
+## Necessary skills
+Understanding of CMake; GitHub; GitHub actions; codecov; clang-tidy and clang-format
+
+## Mentors
+* [Oksana Shadura](mailto:Oksana.shadura@cern.ch)
+* [Vassil Vassilev](mailto:vvasilev@cern.ch)
+
+## Links
```

Comment:
I think this list would be nicer if organized as a list (with `*` in front of each item) …